plugins/org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/RuntimeProjectDescriptor.xtend

```diff
@@ -284,12 +284,10 @@ class RuntimeProjectDescriptor extends TestedProjectDescriptor {
 							}
 							
 							// outline API
-							fragment = adapter.FragmentAdapter {
-								fragment = outline.OutlineTreeProviderFragment auto-inject {}
-							}
-							fragment = adapter.FragmentAdapter {
-								fragment = outline.QuickOutlineFragment auto-inject {}
-							}
+							fragment = ui.outline.OutlineTreeProviderFragment2 {}
```

Comment:
Thanks for that hint ... got lost unintentionally. One more thing @spoenemann need to tell me about ;-)